docs/content.md

```diff
@@ -91,10 +91,10 @@ Atomizer parses any text document you define in your `atomizer.config.js` config
 
 ```html
 <div class="D(f) Jc(sb)">
-    <div class="W(100px) H(100px) Lh(100px) Fz(30px) Ta(c) Bgc(#0280ae.5)">Box 1</div>
```

Comment:
we could look at adding var opacity support for  atomizer.

Reply:
what would the syntax look like for that?

Reply:
`Bgc(--text-foo.5)`

Reply:
Seems kind of odd to add the value onto the variable, versus just making it a variable.